src/System/Net/Benchmarks/Common/Shared/MetricName.cs

```diff
@@ -18,4 +18,5 @@ public static class MetricName
     public const string Rps = "/rps";
     public const string Errors = "/errors";
     public const string Handshake = "/handshake";
+    public const string ConnectionEstablishment = "/connection-establishment";
```

Comment:
Nit: I think it might be better to unify scenario (and metric) name with the job name: connection establishment -> connect